### DISCUSSION TITLE
peak filtering

### DISCUSSION THREAD

**Discussion by AnjaliC4** [GreenleafLab/ArchR#1281] [Questions / Documentation] [answered]
Before running getMarkerFeatures() with wilcox, how can I filter peaks present in more than x% of cells in a cluster?  I know this can be done by exporting peak matrix and doing analysis after - but is there a way to do this in ArchR?

In addition, is it possible to specify peak as "distal" or "promoter" in getMarkerFeatures() ? 

Thank you!

**Comment by rcorces** [comment, marked as answer]
> Before running getMarkerFeatures() with wilcox, how can I filter peaks present in more than x% of cells in a cluster? I know this can be done by exporting peak matrix and doing analysis after - but is there a way to do this in ArchR?

We dont provide any mechanism to do this. The tricky part is deciding what counts as a peak. MACS2 peak calls are notoriously heterogenous so my recommendation would be to create a cutoff of normalized peak signal that you believe is correct in your dataset. ArchR doesnt provide this infrastructure so you'll have to do it yourself from the peak matrix

>  is it possible to specify peak as "distal" or "promoter" in getMarkerFeatures() ?

No but you can subset your peak set and recalculate the peak matrix based on the annotations provided in the peak set as desired.
